### PR TITLE
Create wysiwyg.html

### PR DESCRIPTION
**Pull request by hzik**
# wysiwyg

This KC custom element is based on the https://quilljs.com/ rich text editor.
It stores content in JSON format, so it's omni-channel friendly :)

You can test it by configuring https://amend.cz/wysiwyg/wysiwyg.html url for your custom element.

screenshot

For RTL support specify the Parameters {JSON} part of the configuration with:
{
    "rtl": true
}

### CODE REVIEW

File: WYSIWYG-Quill/wysiwyg.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Wysiwyg</title>
+  <!-- Include jQuery library -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/katex.min.css" />
+  <link rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/monokai-sublime.min.css" />
+  <link rel="stylesheet" href="https://amend.cz/wysiwyg/quill.snow.css" />
```
Comment (Simply007) [inline WYSIWYG-Quill/wysiwyg.html]: Please use the official source defined in [documentation](https://quilljs.com/docs/download/)

```suggestion
  <link href="//cdn.quilljs.com/1.3.6/quill.snow.css" rel="stylesheet">
```

File: WYSIWYG-Quill/wysiwyg.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+      </div>
+
+    </section>
+  </div>
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/katex.min.js"></script>
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
+
+  <script src="https://amend.cz/wysiwyg/quill.min.js"></script>
```
Comment (Simply007) [inline WYSIWYG-Quill/wysiwyg.html]: Please use the official source defined in [documentation](https://quilljs.com/docs/download/)

```suggestion
  <script src="//cdn.quilljs.com/1.3.6/quill.min.js"></script>
```

**Comment by hzik** [comment]
- the min-height adjusted
- image base64 option removed

**Comment by Simply007** [comment]
* added the correct protocol for the uilld editor css and js (incorrectly suggested)
* set min-height for the editor to 150px to prevent overlay on the bottom when opening test formatting
image

**Pull request merged**